osclib/core.py

```diff
@@ -316,7 +316,7 @@ def attribute_value_load(apiurl, project, name, namespace='OSRT'):
     if not len(value):
         return None
 
-    return value[0]
+    return str(value[0])
```

Comment:
Still like this will be almost everywhere (anything returned from API), is there a better general solution for this?